wordle-web/src/views/WordleView.vue

```diff
@@ -1,4 +1,7 @@
 <template>
+  <div class="d-flex justify-end mb-6">
+    <v-btn @click="nameDialog = !nameDialog">{{ displayName }}</v-btn>
```

Comment:
The display name does not get saved to local storage. For example, if you set your name and then go to checkout the leaderboard and come back to play, the name is not saved. I was a little confused on how he wanted us to implement this but my partner solved it using cookies and he said it wasn't too hard. Just an idea.

Reply:
I completely spaced that requirement.  Thanks for the suggestion, I'll look into it.